app/seller/seller-form.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+    View,
+    Text,
+    TextInput,
+    Button,
+    TouchableOpacity,
+    Image,
+    ScrollView,
+} from "react-native";
+import { useForm, useFieldArray, Controller } from "react-hook-form";
+import { useState } from "react";
+import * as ImagePicker from "expo-image-picker";
+import { PostFormSchema, PostFormType } from "@/types/post";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { PostTagsEnum } from "@/enums/post";
+import { MultiSelect } from "react-native-element-dropdown";
+
+const SellerFormPage = () => {
```

Comment:
lets not use this notation for creating React components. lets do named functions [refer](https://clouddevs.com/javascript/named-vs-anonymous-functions/#:~:text=Named%20functions%20shine%20in%20scenarios,requiring%20local%20scope%20and%20closures.)

change to
```tsx
function SellerFormPage() {
```